README.md

```diff
@@ -91,6 +91,7 @@ develop will they get merged into `main`.
 
     ```bash
     cd substrate-docs/
+    nvm install
```

Comment:
```suggestion
    yarn install
```

Reply:
@NukeManDan Yeah I have `nvm` installed on my machine but it didn't auto install missing versions based on [.nvmrc](https://github.com/substrate-developer-hub/substrate-docs/blob/main/.nvmrc) (unless `nmv install` is run explicitly).